docs/cmd/kumactl/HELP.md

```diff
@@ -218,7 +219,7 @@ Flags:
       --admission-server-tls-key string     TLS key for the admission web hooks implemented by the Kuma Control Plane
       --control-plane-image string          image of the Kuma Control Plane component (default "kong-docker-kuma-docker.bintray.io/kuma-cp")
       --control-plane-service-name string   Service name of the Kuma Control Plane (default "kuma-control-plane")
-      --control-plane-version string        version shared by all components of the Kuma Control Plane (default "latest")
+      --control-plane-version string        version shared by all components of the Kuma Control Plane (default "0.2.2-48-ga7b73e9")
```

Comment:
Should be `latest`, CI build will insist on it.

Could you please re-run `make check` and see if it brings `latest` back ?

Otherwise, there must be some issue with `make check` that I need to fix.

Reply:
Done